code/game/gamemodes/setupgame.dm

```diff
@@ -152,10 +152,6 @@
 /proc/setupcult()
 	var/random_cult = pick(all_cults)
 	var/picked_cult = new random_cult() //this seems redundent as fuck
-	log_startup_progress("Summoning eldritch horrors...")
 	if(!picked_cult)//shouldn't happen BUT JUST TO BE SURE LET US KNOW
-		log_startup_progress(" Failed to select a cult datum, Shank a coder!")
-	else
-		log_startup_progress(" [picked_cult] has risen!")
-	//todo:add adminonly datum var, check for said var here...
+		warning(" Failed to select a cult datum, Shank a coder!")
 	return picked_cult
```

Comment:
This doesn't actually fix the problem - the **problem** is that `setupcult` is being called like 10 times when it shouldn't be.